PyInstaller/loader/rthooks/pyi_rth_gstreamer.py

```diff
@@ -16,7 +16,7 @@
 # causes 100% CPU load. (Tested on OSX.)
 os.environ['GST_REGISTRY_FORK'] = 'no'
 
-os.environ['GST_PLUGIN_PATH'] = '{};{}'.format(
+os.environ['GST_PLUGIN_PATH'] = '{}:{}'.format(
     sys._MEIPASS, os.path.join(sys._MEIPASS, 'gst-plugins'))
```

Comment:
Okay, given that the docs explicitly say "colon-separated list of paths", using a colon here should  okay. But we should get of thus useless `format`